app/controllers/admin/countries_controller.rb

```diff
@@ -4,7 +4,7 @@ class Admin::CountriesController < AdminController
   # GET /countries
   # GET /countries.json
   def index
-    @countries = Country.all
+    @countries = Country.all.order(:name)
```

Comment:
No need for "all" in here. Just use `Country.order(:name)`

More about it here: http://guides.rubyonrails.org/active_record_querying.html#ordering